### DISCUSSION TITLE
VSCode Extension error

### DISCUSSION THREAD

**Discussion by SUN7870** [community/community#137728] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I'm trying to install an extension in vscode, but I'm getting an error like "The editor could not be opened due to an unexpected error: Failed to fetch". I'd appreciate your help.
![스크린샷 2024-09-04 220345](https://github.com/user-attachments/assets/595aee82-7eb0-4b27-90ab-10a639bb14fc)

**Comment by xsxo** [comment]
reinstall vacode without delete all files “settings.json” file from appdata 

and try again with trun on cloudflare before install package
